Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.21-alpine3.17
```

Comment:
No uséis versiones arbitrarias del sistema operativo. Como os dije en clase (igual no a ti), cuando se despliega a producción a veces interesa tener una imagen que sea siempre exactamente igual, sin cambios por debajo. A nosotros, sin embargo, que lo vamos a usar para test, escoger una versión de la imagen base del sistema operativo no tiene mucho sentido.
Considera también la versión del lenguaje que estás usando. La política de Go es soportar dos minors atrás, es decir, que tiene una especie de política LTS. No sé si tiene una versión de desarollo. Y esta no es la versión estable, porque estás usando una etiqueta específica, no la etiqueta específica para versión estable.
En resumen: es mejor que te limites en esta imagen a etiquetas semánticas o a versiones LTS que sepas que son la más antigua en la que va a funcionar tu aplicación.

Reply:
Perdone pero no entiendo muy bien como establecer una etiqueta semántica para la imagen de Go. Dentro de los tags permitidos solamente están las siguientes versiones:
![image](https://github.com/pluque01/CofreSagradoVirtual/assets/45895450/66d90438-d494-49b4-a642-62024157cd9c)
Podría usar la versión `latest` de go, pero eso no me aseguraría estar en la más antigua en la que va a funcionar mi aplicación.

Reply:
La más antigua que se soporta ahora es la 1.20; puede que esta imagen no la tenga etiquetada como "lts", pero puede que otras si (o no, no lo sé) En cuanto a la imagen de desarrollo, algunos lenguajes tienen `nightly` o `dev` como etiqueta semántica. Estas, pues no tanto.
En todo caso, te recuerdo que no estás limitado a usar las imágenes oficiales. Puedes usar la imagen de un SO y construir tu propia imagen, usando las releases del lenguaje que te apetezcan.

Reply:
> La más antigua que se soporta ahora es la 1.20; puede que esta imagen no la tenga etiquetada como "lts", pero puede que otras si (o no, no lo sé)

¿Entonces me recomienda que use la versión 1.20 del lenguaje?

Reply:
Aclarado en clase